DataTable/Renderer/TwigRenderer.php

```diff
@@ -3,12 +3,13 @@
 namespace CrossKnowledge\DataTableBundle\DataTable\Renderer;
 
 use CrossKnowledge\DataTableBundle\DataTable\Table\AbstractTable;
+use Twig\Environment;
 
 class TwigRenderer implements RendererInterface
 {
     protected $twig;
 
-    public function __construct(\Twig_Environment $env)
+    public function __construct(Environment $env)
```

Comment:
```suggestion
    /**
     * TwigRenderer constructor
     *
     * @param Environment $env
     */    
    public function __construct(Environment $env)
```
>Add PHPDoc blocks for all classes, methods, and functions (though you may be asked to remove PHPDoc that do not add value);

https://symfony.com/doc/3.4/contributing/code/standards.html#documentation
Also present in 4.4 version: https://symfony.com/doc/4.4/contributing/code/standards.html#documentation

Reply:
Done.